locales/sv_SE/pcsx2_Main.po

```diff
@@ -2411,17 +2409,17 @@ msgstr "Skapa ett nytt Minneskort och anvisa den till denna Sockel."
 msgid "Delete memory file?"
 msgstr "Kassera minnesfil?"
 
+#: pcsx2/gui/Panels/MemoryCardListPanel.cpp:710
+msgid "Failed: Can only duplicate an existing card."
+msgstr "Misslyckades: Kan endast dubblera ett befintligt kort."
```

Comment:
```suggestion
msgstr "Misslyckades: Kan endast duplicera ett befintligt kort."
```

Reply:
"dubblera" is truer swedish than "duplicera", yes?

Reply:
No, "dubblera" means "to double" while "duplicera" means "to duplicate". Most of the time they can be used interchangeably but in this case the difference is important. You're not doubling a memory card, that doesn't even make sense. You're duplicating it, making a copy of it.